### ISSUE TITLE
Does brave now support NFC authentication on iOS 13? (Yubikey 5 NFC for example)

### ISSUE THREAD

**Issue by Nexulo** [brave/brave-ios#1609]
I'm not sure but i read several articles about the fact that brave is the first browser which supports the authentication over NFC on iOS 13? 

I tested that but it's not working, does someone know more about that? Will there come an update for brave?

Best regards
maisen20

**Comment by Nexulo** [comment]
Is there someone working on or are there any news about that? Sorry but don't know how that works here on github.

**Comment by jumde** [comment]
@maisen20 - NFC integration is in progress. Webauthn/U2F using lightning keys should work fine.

**Comment by Nexulo** [comment]
> @maisen20 - NFC integration is in progress. Webauthn/U2F using lightning keys should work fine.

USB-C keys (YubiKey 5C Nano) on iPad Pro do not work, is that right? Is the integration of this in progress too?

**Comment by jumde** [comment]
That is correct, we are tracking that as a separate issue: https://github.com/brave/brave-ios/issues/2019

**Comment by anthonypkeane** [comment]
Moving milestone to 1.14.2

**Comment by kjozwiak** [comment]
It looks like iPhone 6s & 6s+ only support Payments and don't support Read/Writes as per https://www.bluebite.com/nfc/iphone-nfc-compatibility. I checked a few other sources and basically found the same information. I'll go through the lighting port Yubico keys to make sure they're still working. @srirambv will go through NFC has he as an iPhone that supports it.

**Comment by kjozwiak** [comment]
Prerequisite: using a Yubico key with lighting connector with `FW 5.2.3`

Verification **PASSED** on `iPhone 6s+` running `iOS 12.4.1` using `1.14.2 (19.12.11.21)`

* went through `https://demo.yubico.com/webauthn` and ensured the following
   * received the correct animation screen when the Yubico key is disconnected and inserted
   * ensured that quickly disconnecting/connecting the Yubico key doesn't cause any issues/crashes
   * completed the process several times without any issues
   * ensured cancelling/retrying several times doesn't crash/reproduce https://github.com/brave/brave-ios/issues/1388
* ensured that `https://webauthn.io` worked without any issues
* ensured that `https://mdp.github.io/u2fdemo/` worked without any issues
* ensured that `https://twitter.com/` worked without any issues
* ensured that `https://github.com/` worked without any issues

Verification **PASSED** on `iPad Mini 4` running `iOS 12.4.1` using `1.14.2 (19.12.11.21)`

* went through `https://demo.yubico.com/webauthn` and ensured the following
   * received the correct animation screen when the Yubico key is disconnected and inserted
   * ensured that quickly disconnecting/connecting the Yubico key doesn't cause any issues/crashes
   * completed the process several times without any issues
   * ensured cancelling/retrying several times doesn't crash/reproduce https://github.com/brave/brave-ios/issues/1388
* ensured that `https://webauthn.io` worked without any issues
* ensured that `https://mdp.github.io/u2fdemo/` worked without any issues
* ensured that `https://twitter.com/` worked without any issues
* ensured that `https://github.com/` worked without any issues
-------------
Verification passed on iPhone XR with iOS 13.3 running 1.14.2(19.12.12.15)
* Verified scan device popup shows when a security key is required
* Verified GH Login works with security key
* Verified https://demo.yubico.com/webautn/ works fine
* Verified passwordless login works fine and able to scan the device to login
